<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>part8_menu_option_page</class>
 <widget class="QWidget" name="part8_menu_option_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="root_layout">
     <item>
      <widget class="QFrame" name="stickyMenuBar">
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="sticky_layout">
          <item>
           <widget class="QPushButton" name="btnBackToStorePage">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>←</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sticky_menu_name">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>메뉴 이름</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnShare">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>공유</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_option">
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>380</width>
          <height>578</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="layout_menu_info">
           <item>
            <widget class="QLabel" name="label_menu_img">
             <property name="text">
              <string>메뉴 이미지</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_menu_name_2">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>메뉴 이름</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_menu_info">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
        color: lightgray;
       
    }

</string>
             </property>
             <property name="text">
              <string>메뉴 설명</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnShowNutrition">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
        background-color: lightgray;
        border-radius: 8px;
    }


</string>
             </property>
             <property name="text">
              <string>알레르기/영양성분 정보 보기</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <layout class="QVBoxLayout" name="layout_option"/>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="layout_price">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>가격</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_menu_price">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>메뉴 가격</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="layout_count">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>수량</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_abstract">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 1px solid #ccc;
    border-radius: 13px;   /* 반지름이 버튼 크기의 절반이면 완전한 원형 */
    min-width: 26px;       /* 버튼의 최소 가로 크기 */
    min-height: 26px;      /* 버튼의 최소 세로 크기 */
    font-weight: bold;
}</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_menu_count">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_add">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 1px solid #ccc;
    border-radius: 13px;   /* 반지름이 버튼 크기의 절반이면 완전한 원형 */
    min-width: 26px;       /* 버튼의 최소 가로 크기 */
    min-height: 26px;      /* 버튼의 최소 세로 크기 */
    font-weight: bold;
}</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_sticky">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInputOrder">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
        background-color: #00A9FF;
        color: white;
        font-weight: bold;
        border-radius: 8px;
        padding: 10px 20px;
    }
    QPushButton:hover {
        background-color: #0088CC;
    }

</string>
       </property>
       <property name="text">
        <string>배달 카트에 담기</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
